.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: ruby
 rvm:
-  - 2.3.4
-  - 2.4.1
+  - 2.3.6
+  - 2.4.3
+  - 2.5.0
   - ruby-head
-before_install: gem update bundler
+before_install:
+  - gem update --system
```

Comment:
I'm against to update rubygems. ( I understood that is the workaround with 2.5.0. )